src/main/java/org/broadinstitute/hellbender/tools/exome/ReadCountCollectionUtils.java

```diff
@@ -540,12 +547,16 @@ public static ReadCountCollection removeColumnsWithExtremeMedianCounts(final Rea
         if (columnsToKeep.isEmpty()) {
             throw new UserException.BadInput("No column count left after applying the extreme counts outlier filter");
         } else if (columnsToKeep.size() == columnNames.size()) {
-            logger.info(String.format("No column dropped due to extreme counts outside [%.10f, %.10f]", bottomExtremeThreshold, topExtremeThreshold));
+            logger.info(String.format("No column dropped due to extreme counts outside [%.10f, %.10f]",
+                    bottomExtremeThreshold, topExtremeThreshold));
             return readCounts;
         } else {
-            logger.info(String.format("Some columns dropped, %d of %d, as they are classified as having extreme median counts across targets outside [%.10f, %.10f]: e.g. %s",
-                    columnsToDrop.size(), columnNames.size(), bottomExtremeThreshold, topExtremeThreshold,
-                    columnsToDrop.entrySet().stream().limit(10).map(kv -> kv.getKey() + " (" + kv.getValue() + ")").collect(Collectors.joining(", "))));
+            final double droppedPercentage = ((double)(columnsToDrop.size()) / columnNames.size()) * 100;
+            logger.info(String.format("Some columns dropped (%d out of %d, %.2f%%) as they are classified as having extreme " +
+                    "median counts across targets outside [%.10f, %.10f]: e.g. %s",
+                    columnsToDrop.size(), columnNames.size(), droppedPercentage, bottomExtremeThreshold, topExtremeThreshold,
+                    columnsToDrop.entrySet().stream().limit(10).map(kv -> kv.getKey() + " (" + kv.getValue() + ")")
```

Comment:
I vote for the (probably) slightly slower stream version -- it looks easier to read, and that's an appreciable factor.